### PR TITLE
feat: [DHIS2-11594] implement comments in stages&events page

### PR DESCRIPTION
**Pull request by huocha**
### Feature:
Add new column in stage list with icon and number of comments

Screenshot 2021-08-13 at 07 06 54

### CODE REVIEW

File: src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/StageDetail/Comments.component.js [hunk 1]
```diff
@@ -0,0 +1,32 @@
...
+    wrapper: {
+        display: 'flex',
+        alignItems: 'center',
+    },
+    text: {
+        paddingLeft: '2px',
+    },
+};
+export const CommentsPlain = ({ event, classes }: Props) => {
+    const nbComments = event.notes?.length;
```
Comment (simonadomnisoru) [inline src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/StageDetail/Comments.component.js:22]: Hey,
Does the first part `nb` of the`nbComments` have any meaning? 
Thanks!

**Comment by simonadomnisoru** [comment]
Hey @huocha,
Can you add some Cypress tests for this feature? 
Thanks!

File: src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/StageDetail/Comments.component.js (hunk 1 above)
Comment (huocha) [reply src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/StageDetail/Comments.component.js:22]: it's number of comments

**Comment by huocha** [comment]
hey @simonadomnisoru, in order to test this feature someone needs to change the data in testing instance and I don't know how to do that (can only do locally). maybe @JoakimSM can help me adding the comments? Thanks

File: src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/StageDetail/Comments.component.js (hunk 1 above)
Comment (JoakimSM) [reply src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/StageDetail/Comments.component.js:22]: I think a good rule of thumb is: prefer writing the entire word(s) unless the abbreviation is more readable and immediately understandable to other developers. 

Suggestion for this particular variable name: `commentsCount`

In general, we should aim for descriptive variable names to improve readability and maintainability.

Comment (huocha) [reply src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/StageDetail/Comments.component.js:22]: in this case I think it's quite obvious that it points to the total of comments

Comment (JoakimSM) [reply src/core_modules/capture-core/components/WidgetStagesAndEvents/Stages/Stage/StageDetail/Comments.component.js:22]: Let's put this on the agenda for our meeting next week 🙂

**Review by JoakimSM — APPROVED** [review]

**Pull request merged**